.travis.yml

```diff
@@ -0,0 +1,60 @@
+language: python
+python:
+  - "2.7"
+  - "3.5"
+
+env:
+  global:
+    # h5py: pip install work-around for "mpi.h: No such file or directory"
+    - C_INCLUDE_PATH=/usr/lib/openmpi/include
+  matrix:
+    - USE_ADIOS=OFF
+  # - USE_ADIOS=SERIAL
+  # - USE_ADIOS=MPI
+
+addons:
+  apt:
+    packages:
+      - openmpi-bin
+      - libopenmpi-dev
+      - libhdf5-openmpi-dev
+    # - libadios-dev
+
+cache:
+  apt: true
+  directories:
+    - $HOME/.cache/pip
+
+install:
+  - pip install --upgrade pip
+  - pip install --upgrade pyflakes
+# - pip install --upgrade flake8
+# - pip install --upgrade pylint
+  - pip install --upgrade -r requirements.txt
+  - if [ "$USE_ADIOS" == "SERIAL" ]; then pip install --upgrade adios; fi
+  - if [ "$USE_ADIOS" == "MPI" ]; then pip install --upgrade adios_mpi; fi
+
+before_script:
+  # print interesting versions
+  - python --version
+  - ipython --version
+  - python -c "import numpy as np; print( np.__version__ )"
+  - python -c "import h5py as h5; print( h5.__version__ )"
+  - python -c "import mpi4py; print( mpi4py.__version__ )"
+  # static code analysis
+  #   pyflakes: mainly warnings, unused code, etc.
+  # - python -m pyflakes .
```

Comment:
As soon as the current errors documented in #19 are fixed we comment in this line.